Cargo.toml

```diff
@@ -33,10 +34,11 @@ prost-build = "0.10"
 tonic-build = "0.7"
 
 [features]
-default = ["extensions", "remote_storage", "variable_transfer", "registry", "policy_module", "instant_server"]
+default = ["extensions", "remote_storage", "variable_transfer", "registry", "policy_module", "instant_server", "magic_mount"]
```

Comment:
naming: after a second thought (for a week or so) I'm thinking maybe we should call this `key_macro`?

Reply:
`storage_macro`